Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
     mysql2 (0.5.3)
     net-scp (2.0.0)
       net-ssh (>= 2.6.5, < 6.0.0)
     net-ssh (5.2.0)
     netrc (0.11.0)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    oj (3.10.0)
+    oj (3.10.5)
```

Comment:
### [oj](http://www.ohler.com/oj/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.10.0 → 3.10.5

[_[change-log](https://github.com/ohler55/oj/blob/master/CHANGELOG.md), [source-code](https://github.com/ohler55/oj)_]





<details>
<summary>Commits</summary>

A change of **21** commits. See the full changes on [the compare page](https://github.com/ohler55/oj/compare/v3.10.0...v3.10.5).

These are the first 10 commits:
- (6d26584) [Fixes #368 second time around](https://github.com/ohler55/oj/commit/6d265845ba698c5e2035d0f8300ebf6afbd218f7)
- (f969933) [Case insensitve compare for bigdecimal encoding](https://github.com/ohler55/oj/commit/f969933b71577cb4fbf3022eaf736f90fa4469b4)
- (5f372ab) [Update travis.yml](https://github.com/ohler55/oj/commit/5f372ab2f0ab8ce55a262434357c34b10801e5ee)
- (32e481c) [Candidate fix for rails time format issue](https://github.com/ohler55/oj/commit/32e481cfa5be6c4c721949490f7ce58197a2fff1)
- (3ecec05) [remove debug line](https://github.com/ohler55/oj/commit/3ecec05a4397df65f24a44536beaed7ad64c9359)
- (0d786fa) [Update for Ruby 2.7.0](https://github.com/ohler55/oj/commit/0d786fa647fe61487c826ec2daabd652edb20ea8)
- (c32b9ca) [travis update](https://github.com/ohler55/oj/commit/c32b9ca307a5cb44e2dfcf968a8c3e8a59599361)
- (eedad72) [Rails times working better](https://github.com/ohler55/oj/commit/eedad722fff98837248cab6810fe4394ef7f3865)
- (5742832) [Work around Active Support hack](https://github.com/ohler55/oj/commit/574283268a1f6d7a7f07b6e939001a912766cd18)
- (48170be) [Looking for a ruby 2.6 that travis supports](https://github.com/ohler55/oj/commit/48170be77026a1fdd4e805b64540073fca077b85)


</details>